BUILD

```diff
@@ -497,6 +497,12 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
Wondering if we should comment the purpose of creating this target?

Reply:
added comments to the header, should be enough